composer.json

```diff
@@ -20,18 +20,25 @@
         }
     ],
     "require": {
-        "php": ">=8.0",
+        "php": "^8.1 || ^8.2",
         "ext-json": "*",
-        "laravel/framework": "^8.12 || ^9.0 || ^10.0"
+        "laravel/framework": "^9.0 || ^10.0"
     },
     "require-dev": {
-        "orchestra/testbench": "^6.0 || ^7.0 || ^8.0",
-        "tmarsteel/mockery-callable-mock": "~2.1"
+        "orchestra/testbench": "^7.0 || ^8.0",
+        "tmarsteel/mockery-callable-mock": "~2.1",
+        "netsells/code-standards-laravel": "dev-feature/php8.2"
```

Comment:
Will you update this after https://github.com/netsells/code-standards-laravel/pull/1 has been merged/released?

Reply:
That has been the plan @tomchkk - just released and updated! :)